base_tier_validation/models/tier_validation.py

```diff
@@ -672,12 +680,16 @@ def _rejected_tier(self, tiers=False):
                 rec._notify_rejected_review()
 
     def _notify_created_review_body(self):
-        return _("A record to be reviewed has been created by %s.") % (
-            self.env.user.name
+        return self.env._(
+            "A record to be reviewed has been created by %(user_name)s.",
+            user_name=self.env.user.name,
         )
 
     def _notify_requested_review_body(self):
-        return _("A review has been requested by %s.") % (self.env.user.name)
+        return self.env._(
+            "A review has been requested by %(user_name)s.",
+            user_name=self.env.user.name,
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg